src/main/java/au/org/emii/netcdf/iterator/reader/NetcdfReader.java

```diff
@@ -76,4 +80,13 @@ public String getString(Set<IndexValue> indexValues) {
         }
     }
 
+    public String getDisplayValue(Set<IndexValue> indexValues) {
```

Comment:
This accessor is different from all the other ones that are just doing type casting. This one is good because it's got a purpose (generating the value in a form that it can be used for display). But all the other ones that are just type casting are horrible - might as well just write it all in bash.